store/postgres/src/entities.rs

```diff
@@ -773,6 +773,7 @@ pub fn delete_all_entities_for_test_use_only(
         delete from subgraphs.ethereum_contract_data_source_template;
         delete from subgraphs.ethereum_contract_data_source_template_source;
         delete from subgraphs.ethereum_contract_event_handler;
+        delete from subgraphs.subgraph_error;
```

Comment:
Is this necessary? There's an `on delete cascade` on the foreign key `subgraph_error.subgraph_id`

Reply:
Oh that's why we could get away without it. Removed it.